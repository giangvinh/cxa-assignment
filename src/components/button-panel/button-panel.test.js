import React from 'react';
import { shallow } from 'enzyme';
import ButtonPanel from './button-panel.component';

it('should render Button Panel component', () => {
  expect(shallow(<ButtonPanel />)).toMatchSnapshot();
});
