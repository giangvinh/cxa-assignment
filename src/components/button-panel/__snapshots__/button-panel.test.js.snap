// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button Panel component 1`] = `ShallowWrapper {}`;
